LIPS/lip-4.md

```diff
@@ -0,0 +1,47 @@
+---
+lip: 4
+title: 72h Aragon Votes
+status: WIP
+author: Victor Suzdalev (kadmil)
+discussions-to: https://research.lido.fi/t/increase-the-dao-voting-duration/1048
+created: 2021-09-28
+updated: -
+---
+
+# 72h Aragon Votes
+
+Currently Aragon votes on Lido DAO are 24h long. This timeframe has proved to be short enough for reaching the quorum even on non-contentios votes. While most routine DAO operations are expected to switch to EasyTracks (vehicle for pre-approved opt-out DAO actions), any custom blockchain action on behalf of DAO would still require the Aragon vote. The proposal is to make Aragon votes longer, allowing more LDO holders to participate.
```

Comment:
> This timeframe has proved to be short enough for reaching the quorum even on non-contentios votes. 

It's harder to reach quorum on contentious votes, generally (aragon quorum is % of votes yes)

Reply:
> pre-approved opt-out

default-pass

Reply:
the point is "even on non-contentions votes reaching the quorum is hard enough"